packages/angular/projects/tx-native-angular-sdk/src/lib/T.decorator.ts

```diff
@@ -1,30 +1,24 @@
 import { Injector } from '@angular/core';
 
-import { ITXInstanceConfiguration } from './interfaces';
 import { TranslationService } from './translation.service';
 
 /**
  * Decorator for using transparently the translation service as a property
  */
-export const T = (str: string, params?: Record<string, unknown>,
-  instanceConfig?: ITXInstanceConfiguration) => (target: any, key: string) => {
-  const injector = Injector.create(
-    {
-      providers: [
-        { provide: TranslationService, useClass: TranslationService },
-      ],
-    },
-  );
+export const T = (
+    str: string,
+    params?: Record<string, unknown>,
+    instanceAlias?: string,
+) => (target: object, key: string) => {
+  const injector = Injector.create({
+    providers: [ { provide: TranslationService, useClass: TranslationService } ],
+  });
   const translationService = injector.get(TranslationService);
 
   Object.defineProperty(target, key, {
     configurable: false,
     get: () => {
-      if (instanceConfig) {
```

Comment:
I wonder why you removed the `instanceConfig` and the mechanism to add an instance. If I understand well, this way you need to create the instance (`addInstance`) beforehand, correct?